test/components/stories/button_component.stories.json

```diff
@@ -0,0 +1,62 @@
+{
+  "title": "Button Component",
+  "stories": [
+    {
+      "name": "default",
+      "parameters": {
+        "server": {
+          "id": "button_component/default"
+        }
+      },
+      "args": {
+        "type": "primary"
+      },
+      "argTypes": {
+        "type": {
+          "control": {
+            "type": "select"
+          },
+          "name": "Type",
+          "options": [
+            "primary",
+            "secondary",
+            "danger",
+            "link"
+          ]
+        }
+      }
+    },
+    {
+      "name": "primary",
+      "parameters": {
+        "server": {
+          "id": "button_component/primary"
+        }
+      }
+    },
+    {
+      "name": "secondary",
+      "parameters": {
+        "server": {
+          "id": "button_component/secondary"
+        }
+      }
+    },
+    {
+      "name": "danger",
+      "parameters": {
+        "server": {
+          "id": "button_component/danger"
+        }
+      }
+    },
+    {
+      "name": "link",
+      "parameters": {
+        "server": {
+          "id": "button_component/link"
+        }
+      }
+    }
+  ]
+}
```

Comment:
Let's add a new line.

Reply:
This file is auto-generated... Should we manually add a new line (there will be more and more files re-generated each time) 🤔